test/forms/ImporterEoriFormProviderSpec.scala

```diff
@@ -23,7 +23,7 @@ class ImporterEoriFormProviderSpec extends StringFieldBehaviours {
 
   val requiredKey = "importerEori.error.required"
   val lengthKey = "importerEori.error.length"
-  val maxLength = 17
+  val maxLength = 14
```

Comment:
think this may need to be 17

Reply:
@lmugnai incorporated the review comments